### PR TITLE
Styles: reposition rich close button

### PR DESCRIPTION
**Pull request by lgeiger**
PR:
bildschirmfoto 2016-03-30 um 11 59 24
master:
bildschirmfoto 2016-03-30 um 11 59 45

### CODE REVIEW

File: styles/hydrogen.less [hunk 1]
```diff
@@ -30,12 +30,11 @@
         }
 
         .rich-close-button {
+            z-index: 0;
```
Comment (n-riesco) [inline styles/hydrogen.less]: Is this what makes it stay above the plot?

File: styles/hydrogen.less [hunk 2]
```diff
@@ -43,7 +42,7 @@
                 line-height: 1.5em;
                 text-align: center;
                 box-shadow: 0 0 2px 0px rgba(0,0,0,0.5);
-                margin: 0;
+                margin: -50% 0 0 -50%;
```
Comment (n-riesco) [inline styles/hydrogen.less]: How about `0 0 0 -50%` so that it doesn't hide the line above?

File: styles/hydrogen.less (hunk 1 above)
Comment (lgeiger) [inline styles/hydrogen.less]: Yes

File: styles/hydrogen.less (hunk 2 above)
Comment (lgeiger) [inline styles/hydrogen.less]: Hmm but then it will look odd. I've got another idea, I'll try that later if it works.

Comment (lgeiger) [inline styles/hydrogen.less]: bildschirmfoto 2016-03-30 um 23 49 42
Do you like this one better?

**Comment by n-riesco** [comment]
It looks better, but it may not work for small images?

**Comment by lgeiger** [comment]
Yes you're right design should follow function. Two more suggestions:
1. similar to old close button:
bildschirmfoto 2016-03-31 um 01 16 37
2. similar to action-panel:
bildschirmfoto 2016-03-31 um 01 16 15

**Comment by n-riesco** [comment]
Both look good.

**Comment by lgeiger** [comment]
OK lets go with number 1

**Comment by n-riesco** [comment]
Is this PR ready to merge?

**Comment by lgeiger** [comment]
@n-riesco yes

**Comment by n-riesco** [comment]
Sorry. I should've merged this PR after merging #232 .

**Pull request merged**